src/web/app/pages-instructor/student-list/__snapshots__/student-list.component.spec.ts.snap

```diff
@@ -0,0 +1,3044 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`StudentListComponent should snap with default fields 1`] = `
+<tm-student-list
+  backendUrl={[Function String]}
+  courseId=""
+  courseService={[Function CourseService]}
+  enableRemindButton="false"
+  httpRequestService={[Function HttpRequestService]}
+  isHideTableHead="false"
+  listOfStudentsToHide={[Function Array]}
+  navigationService={[Function NavigationService]}
+  ngbModal={[Function NgbModal]}
+  router={[Function Router]}
+  sections={[Function Array]}
+  statusMessageService={[Function StatusMessageService]}
+  useGrayHeading={[Function Boolean]}
+>
+  <div
+    class="table-responsive"
+  >
+    <table
+      class="table table-bordered table-striped m-0"
+    >
+      <thead
+        class="thead-gray"
+        ng-reflect-ng-class="[object Object]"
+      >
+        <tr>
+          <th>
+            Photo
+          </th>
+          
+          <th>
+             Section 
```

Comment:
Yes it should actually. This will only be hidden if the course is not divided into sections. (by default we assume it is)

It will be null. Refer to `StudentListComponent should snap with some student list data with no sections`

Reply:
Gotcha 👍